tools/run_local_http_server.py

```diff
@@ -0,0 +1,33 @@
+#  Copyright 2023 Google LLC.
+#  Copyright (c) Microsoft Corporation.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import sys
+from pathlib import Path
+
+from pytest_httpserver import HTTPServer
+
+# Current directory is not a module, so to import `LocalHttpServer`, its path
+# has to be added to `sys.path`. It is done relative to this file's directory.
+# The `flake8` is disabled for this reason.
+sys.path.append(str(Path(__file__).resolve().parent / '../tests/tools/'))
```

Comment:
Are you sure you need `..` if you added `parent`? Or are you looking for `.parent.parent`?

Reply:
`parent` refers to the file's directory. `tools` can be referenced either by `.parents[1]`, or by `../` prefix. Do you have a strong preference of one over another way?

Reply:
The appeal of the pathlib library is to be OS agnostic, so `.parent` is always better than `../` - IIRC the latter does not work on Windows. LGTM as it is now.